src/app/app.component.ts

```diff
@@ -77,7 +77,7 @@ export class AppComponent {
       }
     } else if (this.hasUserOnboarded === true && this.userType === undefined) {
       console.log('New onboarded user to register');
-      this.router.navigateByUrl('/select-user-type');
+      this.router.navigateByUrl('/instructions');
```

Comment:
I think the redirection to instructions page is to be called from create-requests or accept request page, not set here?

Reply:
here it should be  this.router.navigateByUrl('/select-user-type');
I changed it just to see the instructions page.